scss/_patterns_navigation.scss

```diff
@@ -306,7 +309,7 @@ $navigation-hover-opacity: 0.58;
 
   // separator color on small screens
   .p-navigation__link > a::before {
-    @media (max-width: $breakpoint-navigation-threshold) {
+    @media (max-width: #{$breakpoint-navigation-threshold - 1px}) {
```

Comment:
Is this media query needed at all? Border is only visible on small screens, so having background being set on all widths wont matter.